core/Design.md

```diff
@@ -0,0 +1,12 @@
+## Something 
+
+### off-heap index manager
+
+- Design List
+```
+base on off-heap    
+��
+��������allocate a memory block that contains all the entry had been serialized
```

Comment:
Something like kafka , it also could be impl by file system, SEQ Write Action. I think it would be faster as a queue